pkg/config/config_test.go

```diff
@@ -104,26 +110,34 @@ func TestLoadApiConfig(t *testing.T) {
 				SqsEndpoint:           "http://localhost:9324",
 				SqsAwsRegion:          "sqs-aws-region",
 				SqsQueueName:          "cluster-registry-local",
+				SqsBatchSize:          10,
+				SqsWaitSeconds:        5,
+				SqsRunInterval:        30,
 				K8sResourceId:         "k8s-resource-id",
 				ApiTenantId:           "api-tenant-id",
 				ApiClientId:           "api-client-id",
 				ApiClientSecret:       "api-client-secret",
 				ApiAuthorizedGroupId:  "api-authorized-group-id",
+				ApiCacheTTL:           time.Hour,
```

Comment:
what if a cluster is being updated in less than one hour?

Reply:
cache is invalidated regardless of that TTL whenever a cluster is updated - see https://github.com/adobe/cluster-registry/pull/202/files#diff-50ed20548f3f3b8134f39f359ce2415d2103ae46c65506d4cbfb08205add8ae8R117-R122